packages/teleport/src/boot.tsx

```diff
@@ -29,6 +30,13 @@ cfg.init(window['GRV_CONFIG']);
 // use browser history
 history.init();
 
+const enableTelemetry =
+  localStorage.getItem('enable-telemetry') === 'true' ? true : false;
+
+if (enableTelemetry) {
+  instantiateTelemetry();
+}
```

Comment:
Can you do something like (no idea if this is the correct syntax) 

```suggestion
if (enableTelemetry) {
  import('./telementry-boot').then((module) => module.instantiateTelemetry());
}
```

So we're not including 13 telemetry packages in the main JS bundle if it's not needed

Reply:
Interesting idea, however we need these in the package as we want the user to be able to enable it as they see fit without having to rebuild the UI.

Reply:
How is it enabled? You could also import and instantiate it when toggled in the UI

Reply:
Oh sorry you meant to not have it in the main bundle not not included at all in the bundle :) I can give this a try.

It's enabled by running `localStorage.setItem('enable-telemetry', true)` in the console then refreshing the UI.

Reply:
I've left this change out of this PR because I don't want to introduce further changes at this time. I'll mark it down for a follow-up though 👍